lms/djangoapps/branding/tests/test_page.py

```diff
@@ -163,82 +163,86 @@ def test_course_with_prereq(self):
-    def test_course_cards_sorted_by_default_sorting(self):
-        response = self.client.get('/')
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'index.html')
-
-        # by default the courses will be sorted by their creation dates, earliest first.
-        self.assertEqual(context['courses'][0].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][1].id, self.starting_later.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
-
-        # check the /courses view
-        response = self.client.get(reverse('branding.views.courses'))
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'courseware/courses.html')
-
-        # by default the courses will be sorted by their creation dates, earliest first.
-        self.assertEqual(context['courses'][0].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][1].id, self.starting_later.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
-
-    @patch('student.views.render_to_response', RENDER_MOCK)
-    @patch('courseware.views.render_to_response', RENDER_MOCK)
-    @patch.dict('django.conf.settings.FEATURES', {'ENABLE_COURSE_SORTING_BY_START_DATE': False})
-    def test_course_cards_sorted_by_start_date_disabled(self):
-        response = self.client.get('/')
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'index.html')
-
-        # now the courses will be sorted by their announcement dates.
-        self.assertEqual(context['courses'][0].id, self.starting_later.id)
-        self.assertEqual(context['courses'][1].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
-
-        # check the /courses view as well
-        response = self.client.get(reverse('branding.views.courses'))
-        self.assertEqual(response.status_code, 200)
-        ((template, context), _) = RENDER_MOCK.call_args  # pylint: disable=unpacking-non-sequence
-        self.assertEqual(template, 'courseware/courses.html')
-
-        # now the courses will be sorted by their announcement dates.
-        self.assertEqual(context['courses'][0].id, self.starting_later.id)
-        self.assertEqual(context['courses'][1].id, self.starting_earlier.id)
-        self.assertEqual(context['courses'][2].id, self.course_with_default_start_date.id)
+# THESE TESTS HAVE BEEN COMMENTED OUT BECAUSE FIND COURSES PAGE
+# HAS BEEN SWITCHED OVER TO JS AND EDX SEARCH.
+# AS OF NOW EDX SEARCH DOESN'T SUPPORT RESULTS ORDERING.
```

Comment:
That's the problem, there is no more old setting/context. If we are to do that initial courses card load from Backbone by doing empty search, there is no need to also generate courses cards from django.  What these two tests were doing was checking ordering of those cards. 
I believe we should add order_by parameter to search app and test ordering from Jasmine, removing these tests.